src/backend/commands/analyze.c

```diff
@@ -308,7 +308,7 @@ void analyzeStmt(VacuumStmt *stmt, List *relids, int preferred_seg_num)
 	ListCell				*le1 = NULL;
 	int 					successCount = 0, failCount = 0;
 	StringInfoData 			failNames;
-	ResourceOwner owner, oldOwner;
+	ResourceOwner asOwner, oldOwner1;
```

Comment:
if `as` of `asOwner` means `analyzeStmt`,  we'd better add comments or explanation on the both vars.